PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx

```diff
@@ -680,109 +680,146 @@ struct HfCandidateCreatorXic0Omegac0 {
       KFV0.GetMass(massLam, sigLam);
       if (TMath::Abs(massLam - MassLambda0) > lambdaMassWindow)
         continue;
-      registry.fill(HIST("hKFParticleV0Chi2"), KFV0.GetChi2());
-      if (kfUseV0MassConstraint) {
-        KFV0.SetNonlinearMassConstraint(o2::constants::physics::MassLambda);
-      }
-
+      // err_mass>0 of Lambda
+      if (sigLam <= 0)
+        continue;
+      // chi2>0 && NDF>0 for selecting Lambda
+      if ((KFV0.GetNDF() <= 0 || KFV0.GetChi2() <= 0))
```

Comment:
```suggestion
      if ((kfV0.GetNDF() <= 0 || kfV0.GetChi2() <= 0))
```
Please use lowerCamelCase for names of variables.

Reply:
It's done.